modules/widgets/milestone/milestone.php

```diff
@@ -357,8 +360,48 @@ function form( $instance ) {
 			<input id="<?php echo $this->get_field_id( 'min' ); ?>" class="minutes" name="<?php echo $this->get_field_name( 'min' ); ?>" type="text" value="<?php echo esc_attr( $instance['min'] ); ?>">
 		</fieldset>
 
-		<p>
-			<label for="<?php echo $this->get_field_id( 'message' ); ?>"><?php _e( 'Message', 'jetpack' ); ?></label>
+		<fieldset class="jp-ms-data-unit">
+			<legend><?php esc_html_e( 'Time Unit', 'jetpack' ); ?></legend>
```

Comment:
Do we need both of these `Time Unit`s? There are two displayed and it seems unintentional 

![screen shot 2017-09-22 at 2 58 29 pm](https://user-images.githubusercontent.com/7129409/30760052-93c9bb9e-9fa6-11e7-8137-41f7a386820b.png)

Reply:
Oops. Missed committing a CSS file change. This is fixed in https://github.com/Automattic/jetpack/pull/7782/commits/dd4d61ec0b2ce191c2994acc3ee8a4da4ce73120. Thx!